src/components/SearchBox.tsx

```diff
@@ -42,10 +42,18 @@ const SearchBox = ({ setSearchResults }: SearchBoxProps) => {
       } = await axios.get(
         API_URL + `?term=${searchValue}&entity=album&limit=10`
       );
-      setSearchResults(results);
-      setKeyword(searchValue);
+
+      if (results.length === 0) {
+        setHelperText('No results match criteria.');
```

Comment:
https://github.com/e1401/musicolor/assets/38010322/84568c12-48a3-4915-a6a8-786ed7b6f72b


Take a look at this situation where user has searched for something that exists on the api, and then they search for something else and they get a "No results match criteria." message but the previous results remain on screen. In this case previous results should be removed.